actions/st2_pkg_upgrade_deps_el8.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+VERSION=$1
+ENTERPRISE=$2
+SHORT_VERSION=`echo ${VERSION} | cut -d "." -f1-2`
+
+if [ ${SHORT_VERSION} = "2.10" ] && [ ${ENTERPRISE} -eq 0 ]; then
```

Comment:
We don't need the st2 `2.10` upgrade scripting as EL8 will be introduced in `3.2`.

This file is good to keep just empty.

Reply:
@armab So I think what you are asking me to do is to replace the if statements with:

```
if [ ${SHORT_VERSION} = "2.10" ] && [ ${ENTERPRISE} -eq 0 ]; then
```

eg. giving SHORT_VERSION a blank value to keep the logic around? 

Or should I just comment it out?

Reply:
Just pushed a new commit with the updated file.